charts/plugin-health-scoring/values.yaml

```diff
@@ -55,10 +55,17 @@ probes:
 config:
   ucCron: "0 */15 * * * *"
   peCron: "0 10 * * * *"
-  githubOauth: s3cr3tToken
+  github:
+    appId: app-id
+    privateKey: >-
+      -----BEGIN PRIVATE KEY-----
+      blablablabla
+      -----BEGIN PRIVATE KEY-----
+    appInstallationName: jenkinsci
```

Comment:
@alecharp Is it used in the Plugins Health Scoring application?

I didn't see any use of it in https://github.com/jenkins-infra/plugin-health-scoring/pull/159/files

I would remove it if it's not currently used.

Reply:
It is https://github.com/jenkins-infra/plugin-health-scoring/pull/159/files#diff-7d075ccdbf2d33cad45b93580fbd2097c0730d6e79a2ee5cfc89a0800f48085eR94-R111

Reply:
It is here: https://github.com/jenkins-infra/plugin-health-scoring/pull/159/files#diff-7d075ccdbf2d33cad45b93580fbd2097c0730d6e79a2ee5cfc89a0800f48085eR85-R91

Reply:
Noted, thanks.

Looking at this code and variable name like `privateKeyPEM`, does this mean the key doesn't have to be converted from PEM to DER?

Reply:
It doesn't seem so, will see.